### ISSUE TITLE
AutoSklearnClassifier seems to exit application on finishing fit

### ISSUE THREAD

**Issue by jrs53** [automl/auto-sklearn#1029]
[AutoML(1)_fe4182f8674c69a5018ac777ba60abec.log](https://github.com/automl/auto-sklearn/files/5645798/AutoML.1._fe4182f8674c69a5018ac777ba60abec.log)
[autosklearn.log](https://github.com/automl/auto-sklearn/files/5645799/autosklearn.log)

## Describe the bug ##

When I run automl.fit, it seems to take the expected amount of time (specified by time_left_for_this_task), but my application immediately quits with no apparent error messages.

## To Reproduce ##
```
automl = AutoSklearnClassifier(time_left_for_this_task = 2*60)
logging.info("Fitting classifier")
automl.fit(X_train, y_train)
logging.info("Finished fitting classifier")
```
This runs for ~2 mins, I see "Fitting Classifier" in my output, but never "Finished fitting classifier".

This only seems to happen with my data; e.g. load_digits from the Start page is fine.

## Expected behavior ##
I expect the classifier to fit and execution to continue

## Actual behavior, stacktrace or logfile ##
No stacktrace, logs attached

## Environment and installation: ##

Please give details about your installation:

Running inside Docker container (mfeurer/auto-sklearn both master and development) on Amazon Linux

**Comment by jrs53** [comment]
Ahh, might `automl.fit` do something that means subsequent `logging` statements don't actually do what I expect?

**Comment by mfeurer** [comment]
> Ahh, might automl.fit do something that means subsequent logging statements don't actually do what I expect?

Yes, indeed it does. We should make sure to not mess with the default logger. @franchuterivera would it be possible to add this requirement to #1017

**Comment by franchuterivera** [comment]
Hello! I think this issue was solved with #1077, but let me clarify the expectation here:

- The [API](https://automl.github.io/auto-sklearn/master/api.html) has support to provide a logging configuration similar to the [default](https://github.com/automl/auto-sklearn/blob/58e36bea83c30a21872890c7b4b235fd198dcf7b/autosklearn/util/logging.yaml#L27) one used by autosklearn .

Notice that autosklearn by default makes the root logs to console be warning level. If you do `logger.warning('Finisher')` you will be able to see the desired msg. If you do want to see info level, kindly pass a logging yaml file to autosklearn.

Thanks a lot for originally reporting the issue! I am going to close it now, as it seems to be fixed.
